cni/network/multitenancy.go

```diff
@@ -339,55 +319,3 @@ var (
 	errInfraVnet     = errors.New("infravnet not populated")
 	errSubnetOverlap = errors.New("subnet overlap error")
 )
-
-// GetMultiTenancyCNIResult retrieves network goal state of a container from CNS
-func (plugin *NetPlugin) GetMultiTenancyCNIResult(
-	ctx context.Context,
-	enableInfraVnet bool,
-	nwCfg *cni.NetworkConfig,
-	k8sPodName string,
-	k8sNamespace string,
-	ifName string) (res *cniTypesCurr.Result, resp *cns.GetNetworkContainerResponse, prefix net.IPNet, infraRes *cniTypesCurr.Result, err error) {
-
-	result, cnsNetworkConfig, subnetPrefix, err := plugin.multitenancyClient.GetContainerNetworkConfiguration(ctx, nwCfg, k8sPodName, k8sNamespace, ifName)
-	if err != nil {
-		log.Printf("GetContainerNetworkConfiguration failed for podname %v namespace %v with error %v", k8sPodName, k8sNamespace, err)
-		return nil, nil, net.IPNet{}, nil, fmt.Errorf("GetContainerNetworkConfiguration failed:%w", err)
-	}
-
-	log.Printf("PrimaryInterfaceIdentifier :%v", subnetPrefix.IP.String())
-
-	if checkIfSubnetOverlaps(enableInfraVnet, nwCfg, cnsNetworkConfig) {
-		buf := fmt.Sprintf("InfraVnet %v overlaps with customerVnet %+v", nwCfg.InfraVnetAddressSpace, cnsNetworkConfig.CnetAddressSpace)
-		log.Printf(buf)
-		return nil, nil, net.IPNet{}, nil, errSubnetOverlap
-	}
-
-	if nwCfg.EnableSnatOnHost {
```

Comment:
where did you move this?

Reply:
l451 in network.go

Reply:
moved to GetContainerNetworkConfiguration now